src/actions/EditBaleModalActions.js

```diff
@@ -1,22 +1,32 @@
+import BaleController from '../controllers/BaleController';
+
 export const actionTypes = {
   OPEN_EDIT_BALE_MODAL: 'OPEN_EDIT_BALE_MODAL',
-  CLOSE_EDIT_BALE_MODAL: 'OPEN_EDIT_BALE_MODAL',
+  CLOSE_EDIT_BALE_MODAL: 'CLOSE_EDIT_BALE_MODAL',
 };
 
-const openEditBaleModalType = () => ({
+const openEditBaleModalType = bale => ({
   type: actionTypes.OPEN_EDIT_BALE_MODAL,
-  isVisible: true,
+  bale,
 });
 
-export const openEditBaleModal = () => (dispatch) => {
-  dispatch(openEditBaleModalType());
+export const openEditBaleModal = bale => (dispatch) => {
+  dispatch(openEditBaleModalType(bale));
 };
 
 const closeEditBaleModalType = () => ({
   type: actionTypes.CLOSE_EDIT_BALE_MODAL,
-  isVisible: false,
 });
 
 export const closeEditBaleModal = () => (dispatch) => {
   dispatch(closeEditBaleModalType());
 };
+
+export const editBale = (token, bale, weight, material) => async (dispatch) => {
+  try {
+    const { baleData } = await BaleController.editBale(token, bale, weight, material);
+    dispatch(closeEditBaleModal(baleData));
```

Comment:
This one may be up for discussion and to be honest, is not really something i care that much about. This action is closing and editing, which is smart, as you dont want to keep the window open after you edit. But my point is that the modal should close in the editBaleModal.js file when executing line 40. Its a super small detail, but it adds to code clarity. Another way to adress the "issue" (i repeat its not really a problem) is to add a comment on line 40 to clarify that the editBale action closes the modal.

Reply:
Yes and i agree with you. What im saying is when should you close such modal? Should the "edit" action close it or should the "accept edit function" in the component close it? 

It's a small detail for the sake of code quality. In my opinion it should be made at the component or at least clarify it at the component.